src/control/cmd/daos/filesystem.go

```diff
@@ -37,6 +37,7 @@ type fsCmd struct {
 	ResetAttr      fsResetAttrCmd      `command:"reset-attr" description:"reset fs attributes"`
 	ResetChunkSize fsResetChunkSizeCmd `command:"reset-chunk-size" description:"reset fs chunk size"`
 	ResetObjClass  fsResetOclassCmd    `command:"reset-oclass" description:"reset fs obj class"`
+	DfuseEvict     fsDfuseEvictCmd     `command:"evict" description:"Evict object from dfuse"`
```

Comment:
If this is ONLY for dfuse - not DFS - maybe `evict` -> `dfuse-evict`?
Or maybe it's okay if it only makes sense for dfuse anyway

Reply:
I think the only thing al these commands have in common is they're for POSIX containers and therefore DFS at some level.  `evict` doesn't mean anything in dfs directly that I'm aware of.

Reply:
one thing i was thinking about here is that the daos fs options are on the container (server side), but these dfuse commands apply only on the client side (just the node they are running on). so they are just local operations and only executed on the node where this runs.
i don't know if that counts as grounds that we need to split this off into 2 tools or just have an extension to the dfuse command, or just keep it as it is. what do you think?

Reply:
We discussed this extensively in the working group calls, this was your suggestion.

I'd be fine with making a new subcommand for the `daos` command other than `daos filesystem` but I don't want to do it in this PR.

Reply:
I agree though, there's a clear distinction between commands that have an effect on the container itself vs commands that change local in-memory state.